Sources/Client/Utils/Tests.swift

```diff
@@ -8,7 +8,7 @@
 
 import Foundation
 
-func isTests() -> Bool {
+public func isTestsEnvironment() -> Bool {
```

Comment:
I think this should be `open` so users can override it

Reply:
I don't understand, why?